tests/test_iscsi.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2024, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+from drgn_tools import iscsi
+from drgn_tools.module import KernelModule
+
+
+def test_iscsi(prog):
+    # make sure iscsi module is loaded
+    ISCSI_MODS = ["libiscsi", "scsi_transport_iscsi"]
+    for mod in ISCSI_MODS:
+        km = KernelModule.find(prog, mod)
+        if not km:
+            return
```

Comment:
With `ensure_debuginfo` in the beginning of `print_iscsi_report`, above loading debuginfo code is not needed.